status-im/status-react
# [FIX #3651] Change console icon

fixes #3651

### Summary:

Change the Console chat icon to look more like a dev tool.

### Todo:
- [x] change console icon
- [x] remove dapp badge from console icon
- [x] change console name to "Status Console"
- [x] change empty screen text in console chat







### Steps to test:
- Open Status
- Navigate to home screen
- Make sure the icon looks as described in #3651 
- Make sure that the contact name is "Status Console"
- Open the chat with no messages
- Make sure the empty screen text reads `Look under the hood! Console is a javascript runtime environment that exposes the whole web3 API. <br>Type "web3." to get started.` as described in #3651

status: ready
